controllers/healthcheck_controller.go

```diff
@@ -90,23 +90,50 @@ func (r *HealthCheckReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 	ctx := context.Background()
 	log := r.Log.WithValues(hcKind, req.NamespacedName)
 
+	log.Info("Starting HealthCheck reconcile ...", "for this", "hc")
```

Comment:
@RaviHari, can you help me understand the thinking behind using variable number of string params passed to the `Info()` method?

I see you're doing it here, as well as at least lines 104 and 107.

Reply:
It is from an old branch fixed this..